docs/en_US/Release.md

```diff
@@ -1,5 +1,47 @@
 # ChangeLog
 
+## Release 1.5 - 4/13/2020
+
+### New Features and Documentation
+
+#### Hyper-Parameter Optimizing
+
+* New tuner: [Population Based Training (PBT)](https://github.com/microsoft/nni/blob/master/docs/en_US/Tuner/PBTTuner.md)
```

Comment:
better to use the link like this: https://github.com/microsoft/nni/blob/v1.5/docs/en_US/Tuner/PBTTuner.md